component-library/src/app/pages/date-picker-documentation/date-picker-documentation.component.html

```diff
@@ -0,0 +1,661 @@
+                {{ 'General.DontHeading' | translate }}
+              </h5>
+              <p>
+                {{
+                  'DatePicker.DesignGuideLines.ErrorValidationDont' | translate
+                }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="element-wrapper-container">
+        <h5 class="sub-headings">
+          {{ 'DatePicker.DesignGuideLines.Horizontal' | translate }}
+        </h5>
+
+        <div class="row row-container">
+          <div class="column left-column">
+            <img
+              src="assets/img/datePicker-content/horizontal-do.png"
+              alt="{{
+                'DatePicker.DesignGuideLines.HorizontalDoAlt' | translate
+              }}"
+            />
+            <div class="do-dont-container">
+              <h5 class="do-dont-heading text-color-green">
+                {{ 'General.DoHeading' | translate }}
+              </h5>
+              <p>
+                {{ 'DatePicker.DesignGuideLines.HorizontalDo' | translate }}
+              </p>
+            </div>
+          </div>
+          <div class="column right-column">
+            <img
+              src="assets/img/datePicker-content/horizontal-dont.png"
+              alt="{{
+                'DatePicker.DesignGuideLines.HorizontalDontAlt' | translate
+              }}"
+            />
+            <div class="do-dont-container">
+              <h5 class="do-dont-heading text-color-red">
+                {{ 'General.DontHeading' | translate }}
+              </h5>
+              <p>
+                {{ 'DatePicker.DesignGuideLines.HorizontalDont' | translate }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="anatomy">
+    <app-title-slug-url
+      [config]="headingConfig.anatomySlugConfig"
+    ></app-title-slug-url>
+    <img
+      class="anatomy-image-container"
+      src="assets/img/datePicker-content/anatomy.png"
+      alt="{{ 'DatePicker.Anatomy.ImgAlt' | translate }}"
+    /><br />
+    <ng-container *ngFor="let content of anatomyContentItems; let i = index">
+      <div class="anatomy-content-container">
+        <div class="conter-and-heading-container">
+          <span class="round-purple-num-count">{{ i + 1 }}</span>
+          <h5 class="round-purple-num-heading-text">
+            {{ content.title | translate }}
+          </h5>
+        </div>
+        <p class="anatomy-description">
+          {{ content.description | translate }}
+        </p>
+      </div>
+    </ng-container>
+  </section>
+  <section class="specs">
+    <app-title-slug-url
+      [config]="headingConfig.specsSlugConfig"
+    ></app-title-slug-url>
+    <img
+      class="single-img"
+      src="assets/img/datePicker-content/specs.png"
+      alt="{{ 'DatePicker.Specs.ImgAlt' | translate }}"
+    />
+    <p class="specs-description">
+      {{ 'DatePicker.Specs.Text' | translate }}
```

Comment:
Mahsa is leaving a comment on figma for Sarah to determine whether the alt text description should be displayed in the template or as an alt tag on the img given that there is a separate alt text provided in figma. This applies to banner component as well but is not the case for input component